datalad/tests/test_witless_runner.py

```diff
@@ -157,7 +157,7 @@ def test_runner_stdin_no_capture():
     # Ensure that stdin writing alone progresses
     runner = Runner()
     runner.run(
-        py2cmd('import sys; print(sys.stdin.read())'),
+        py2cmd('import sys; _ = sys.stdin.read()'),
```

Comment:
now looking at the test, nothing in it really "Ensure that stdin writing alone progresses" does it?

I guess it could have tried to `swallow_outputs` and analyze them but not sure if that would interfere or not.  May be (if only for ensuring via visual inspection) it should print something at the end, e.g.

```suggestion
        py2cmd('import sys; print(sys.stdin.read()[-10:])'),
```
?

Reply:
Thanks for the suggestion